core/services/bridget/settings.py

```diff
@@ -0,0 +1,44 @@
+from typing import Any, Dict
+
+import pykson  # type: ignore
+from commonwealth.settings import settings
+
+
+class BridgeSettingsSpec(pykson.JsonObject):
```

Comment:
Just realized, this class should also include a "V1" at the end, as it could change in the future, right?

Reply:
Yes, it makes sense.